### ISSUE TITLE
Keyboard focus indicator lost in left rail menu on istio.io/news; unable to open headers

### ISSUE THREAD

**Issue by Lealikeintea** [istio/istio.io#7427]
When navigating through istio.io/news via keyboard using the tab key, the focus indicator is not visible in the closed headings in the left side rail menu, causing the user to lose their place and violating three WCAG accessibility principles (linked below). Additionally, the user is unable to select and open the necessary left side rail menu headers to see the sub-menu options because the focus indicator is not present.

[2.4.3 - Focus Order (Level A) -](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=243#focus-order) If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

[2.4.7 - Focus Visible (Level AA) ](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=243#focus-visible)- Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.

[2.4.8 - Location (Level AAA)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=243#location) - Information about the user's location within a set of Web pages is available.
